### PR TITLE
WMS-676 | Move cellLink to UI.Table

### PR DESCRIPTION
**Pull request by PedroHLC**
#### :thinking: What?
* Move `cellLink` to UI.Table

#### :man_shrugging: Why?
* This code was in the app and may be reused.

#### :pushpin: Jira Issue
[WMS-676](https://paacklogistics.atlassian.net/browse/WMS-676)

#### :no_good: Blocked by
Not blocked.

#### :clipboard: Pending
Nothing.

### CODE REVIEW

File: src/UI/Internal/Tables/View.elm [hunk 1]
```diff
@@ -47,6 +52,26 @@ simpleText renderConfig text =
             ]
 
 
+linkedText : RenderConfig -> Link -> Text -> Element msg
+linkedText renderConfig link text =
+    text
+        |> Text.withOverflow ellipsizeWithTooltip
+        |> Text.withColor (Palette.color tonePrimary brightnessMiddle)
+        |> Text.renderElement renderConfig
```
Comment (renanpvaz) [inline src/UI/Internal/Tables/View.elm:60]: what if we need links with the overflow behavior outside the table? We'd have to copy it over, so maybe we should create a standalone link component outside of the button. Just something to keep in mind

**Review by renanpvaz — APPROVED** [review]

File: src/UI/Internal/Tables/View.elm (hunk 1 above)
Comment (PedroHLC) [reply src/UI/Internal/Tables/View.elm:60]: I find it interesting, could you propose the API for that? Also, CC @jouderianjr

Comment (jouderianjr) [reply src/UI/Internal/Tables/View.elm:60]: yeah, taking in mind the overflow, it makes sense to have it also as a component

**Review by jouderianjr — APPROVED** [review]

File: src/UI/Internal/Tables/View.elm (hunk 1 above)
Comment (jouderianjr) [reply src/UI/Internal/Tables/View.elm:60]: We can do it after the BF thing, can you create a card for that @PedroHLC ?

**Pull request merged**